ext/flash/lib.rs

```diff
@@ -244,12 +243,15 @@ async fn op_flash_write_resource(
         let mut stat: libc::stat = unsafe { std::mem::zeroed() };
         // SAFETY: call to libc::fstat.
         if unsafe { libc::fstat(fd, &mut stat) } >= 0 {
-          let _ = sock.write(
-            format!("Content-Length: {}\r\n\r\n", stat.st_size).as_bytes(),
-          );
+          let cnt_length = format!("Content-Length: {}\r\n\r\n", stat.st_size);
```

Comment:
This is an additional allocation, maybe we could create 3 `UnixIoSlice` instances containing:
- `Content-Length: `
- `{stat.st_size}`
- `\r\n\r\n`

Reply:
Fixed. Now uses `itoa` which uses a stack allocated buffer for formating st_size...we're doing zero heap allocation now.